### ISSUE TITLE
`blockid` shouldn't be converted to lower case in logs

### ISSUE THREAD

**Issue by jiasli** [Azure/azure-storage-azcopy#1609]
##### Note: The version is visible when running AzCopy without any argument
10.13.0

### Which platform are you using? (ex: Windows, Mac, Linux)
Windows

### What command did you run?

```
./azcopy.exe copy "C:\Users\xxx\Downloads\test.msi" "https://test.blob.core.windows.net/con1/test.msi?sv=2020-08-04&se=2021-12-02T13%3A13%3A42Z&sr=c&sp=rwl&sig=xxx" --log-level=DEBUG
```

### What problem was encountered?

In the log file:    

```
PUT https://test.blob.core.windows.net/con1/test.msi?blockid=mwizyzy3ytitzmjimy0wyjq0ltvhzdetotjjngrkmtcwzdk0&comp=block&se=2021-12-02t13%3A13%3A42z&sig=-REDACTED-&sp=rwl&sr=c&sv=2020-08-04&timeout=901
```

`blockid` is base-64 encoded which is **case sensitive**. Its original value is `MWIzYzY3YTItZmJiMy0wYjQ0LTVhZDEtOTJjNGRkMTcwZDk0`. Converting to lower case makes it impossible to decode. Therefore, **`blockid` should not be converted to lower case**.

### How can we reproduce the problem in the simplest way?
Run the command and check log.

### Have you found a mitigation/solution?
No

**Comment by mohsha-msft** [comment]
Hey @jiasli,

Thanks for reaching out to us!
I was able to reproduce this bug. It happens only when SAS is involved in the URL string. 
I'll add this to our backlog and we'll try to fix it in the coming releases.

**Comment by mohsha-msft** [comment]
Fix released in AzCopy `10.14.0`
